tests/system/lab/test

```diff
@@ -24,6 +24,7 @@ display_and_collect_err () {
 
 
 URL="http://$FLEDGE_IP:8081/fledge"
+PROJECT_ROOT=$(git rev-parse --show-toplevel)
```

Comment:
We can achieve this without git utility. 
The special variable __file__contains the path to the current file. From that we can get the directory using either pathlib or the os.path module. For example:
```
1. For the current working directory:

import pathlib
pathlib.Path().resolve()

2. For the directory of the script being run:

import pathlib
pathlib.Path(__file__).parent.resolve()

```

Reply:
correct, this comment is only so that we don't do another copy/paste same at other places.